doc/postgresql.md

```diff
@@ -0,0 +1,18 @@
+# Cloud Resource Operator - PostgreSQL
+
+## Usage
+To seed a Kubernetes/Openshift cluster with an example Postgres resource:
+```
+$ make cluster/prepare 
+$ make cluster/seed/postgres
+```
+
+## Strategy 
+A config map object is expected to exist for each provider that will be used by the operator. This config map contains provider-specific information about how to deploy a particular resource type, such as blob storage. In the Cloud Resources Operator, this provider-specific configuration is called a strategy. An example of the AWS strategy can be seen [here](../deploy/examples/config_aws_strat.yaml)
```

Comment:
Can we add information for each resource about which providers/strategies are supported?